.circleci/config.yml

```diff
@@ -316,11 +316,15 @@ jobs:
             conda install -y anaconda-client ccache cmake git ninja conda-build pip
             echo $(which -a python)
             pip install gitpython
+            cd habitat-sim
+            git submodule update --init --recursive --jobs 8
+            pip install .
+            pip uninstall -y habitat-sim
+            rm -rf build
```

Comment:
This somehow makes `conda build` work on the osx image. We're not exactly sure why.

It adds ~13 minutes to the build stage, but we can live with it for now.
Fortunately, this hack won't be necessary when we move from `setup.py install` to `pip install .`.

Thanks @rpartsey for reproducing and finding the workaround.